ocfweb/docs/docs/staff/getinvolved.md

```diff
@@ -0,0 +1,227 @@
+   * Printer querying and statistics ([printing/][ocflib-printing])
+* [utils][utils] ([issues][utils/issues]) - similar to ocflib, this contains
+  Python scripts that are run by humans.
+   * Paper quota refund ([printing/paper][utils-paper])
+   * Query/search OCF accounts ([acct/][utils-acct])
+   * For users: create MySQL database and HTTP webdir
+     ([makeservices/makemysql-real][utils-makemysql-real],
+     [makeservices/makehttp][utils-makehttp])
+   * Create new virtual machines ([staff/sys/makevm][utils-makevm])
+   * Wake up desktops from sleep ([staff/lab/lab-wakeup][utils-lab-wakeup])
+* [slackbridge][slackbridge] ([issues][slackbridge/issues]) - bridges our IRC
+  network with our Slack organization, so staff members can exercise choice in
+  what chat program they’d like to use. Written in Python.
+* [ircbot][ircbot] ([issues][ircbot/issues]) - a fun chatbot, also used to
+  approve new account requests. Written in Python.
+   * See https://ircbot.ocf.berkeley.edu/ for a listing of IRC bot functionality
+
+#### TODOs
+
+If you’re feeling particularly adventurous, you can [search for the string
+“TODO” across our entire codebase][sourcegraph-todo].
+
+#### Help requests
+
+During the semester, our [[help@ocf.berkeley.edu mailing list|doc contact]]
+gets around 14 emails per week from the UC Berkeley community. These emails get
+responded to by volunteers like you. Most requests are about requesting virtual
+hosting and helping users debug or fix their websites. If you’re interested in
+seeing how this process works, ask a staffer to add you to the RT mailing list.
+Since this is high-volume, we recommend filtering these to a different folder
+so you don’t get notified all the time.
+
+Joining this list doesn’t obligate you to respond to help requests. It’s
+perfectly acceptable to only read tickets as a way to learn more about what we
+do.
+
+If you’re on staff, you can view the ticket archive at rt.ocf.berkeley.edu.
+
+#### Your idea here
+
+As a volunteer organization, the OCF’s direction is driven by the interests of
+our members. If you have a new idea for the OCF, we want to help you build it.
+You can use our chat channels or talk to a veteran staffer in person to get a
+starting point and roadmap.
+
+### Staff Hours
+
+During the week, veteran staffers host “staff hours”
+([ocf.io/staffhours][staffhours]), where we provide support to users of our
+services. As a newbie, attending staff hours can serve multiple purposes:
+
+   * Learn by shadowing veteran staffers as they help people out
+   * Talk about your interests so we can help you find stuff to work on
+   * Get one-on-one mentoring so you can learn about the vast complex world of
+     OCF infrastructure
+   * Get context and more information on a particular area you’re interested
+     in contributing in
+
+### Slack or IRC
+
+OCF staff often communicate on Slack or IRC. For instructions on how to join
+Slack, see [ocf.io/slack][slack]. Some important channels:
```

Comment:
perhaps note that these are bridged together, and maybe link IRC to thelounge?

Reply:
I'll just link to the docs